helm/alfresco-content-services/templates/config-transformer-routes.yaml

```diff
@@ -96,7 +96,19 @@ data:
             {"sourceMediaType": "application/vnd.ms-excel",                                                   "maxSourceSizeBytes": -1, "targetMediaType": "application/vnd.alfresco.ai.features.v1+json" },
             {"sourceMediaType": "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",          "maxSourceSizeBytes": -1, "targetMediaType": "application/vnd.alfresco.ai.features.v1+json" },
             {"sourceMediaType": "application/vnd.ms-powerpoint",                                              "maxSourceSizeBytes": -1, "targetMediaType": "application/vnd.alfresco.ai.features.v1+json" },
```

Comment:
I think you need to remove the ms-powerpoint line as that is now covered by the new transform.